dynamic-plugins/wrappers/backstage-plugin-notifications-backend-dynamic/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "backstage-plugin-notifications-backend",
+  "version": "0.4.1",
+  "main": "./dist/index.cjs.js",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.27.7",
+    "pluginId": "notifications",
+    "pluginPackages": [
+      "backstage-plugin-notifications",
+      "backstage-plugin-notifications-backend"
+    ]
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage/plugin-notifications-backend --embed-package @backstage/plugin-notifications-node --embed-package @backstage/plugin-signals-node",
```

Comment:
```suggestion
    "export-dynamic": "janus-cli package export-dynamic-plugin --shared-package !/@backstage/plugin-notifications/ --shared-package !/@backstage/plugin-signals/ --embed-package @backstage/plugin-notifications-backend --embed-package @backstage/plugin-signals-node",
```

This is the typical complete `export-dynamic` command for a plugin wrapper for a backend plugin **_which is part of the `@backstage` scope_**.

Let me explain it in detail:
-   `--shared-package !/@backstage/plugin-notifications/` means that we should **not** (see the `!` at start) consider packages satisfying the `@backstage/plugin-notifications` regex as being shared packages (== brought by the main Backstage app). This is useful because by default, the `export-dynamic-plugin` command considers all the `@backstage`-scoped packages as shared packages.
- `--embed-package @backstage/plugin-notifications-backend` is the wrapped package, which should be embedded since it very probably depends on some shared `@backstage` packages (like `@backstage/backend-plugin-api`)
- `--embed-package @backstage/plugin-signal-node` is a direct dependency of the wrapped package, which should be embedded since it very probably depends on some shared `@backstage` packages.

cc @PatAKnight

Reply:
Probably at some point we'll have to provide a formal documentation about writing wrappers, and more generally, about how to use the `--shared-package` and `--embed-package`. cc @gashcrumb

Reply:
Thank you for the detailed explanation! I understand it much more clearly now.